ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/persistence/EphemeralH2DatabaseSandboxMain.scala

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.persistence
+
+import com.digitalasset.platform.sandbox.SandboxMain
+
+object EphemeralH2DatabaseSandboxMain extends App {
```

Comment:
This class can be removed. See my comment on `ledger/sandbox/BUILD.bazel`.

Reply:
👍